operations/commands.md

```diff
@@ -256,40 +296,49 @@ Adds a new printer command representing printing of the specified document on a
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../concepts/multi-property.md), ignored otherwise. |
-| `PrinterId` | string | required | Uniqque identifier of the `Printer` [Device](devices.md#device) where to print the document. |
-| `CopyCount` | int | required | Count of copies to be printed. |
+| `EnterpriseId` | string | optional | Unique identifier of the enterprise. Required when using [Portfolio Access Tokens](../concepts/multi-property.md), ignored otherwise. |
+| `PrinterId` | string | required | Uniqque identifier of the Printer [Device](devices.md#device) where to print the document. |
```

Comment:
Typo

Reply:
That's imported from OAS enricher 🤦
I will fix it